index.md

```diff
@@ -118,6 +124,8 @@ Det å bli forelder er en fantastisk opplevelse og vi unner alle å ta en velfor
 Tiden du er i permisjon vil ikke gå utover din ansiennitet i selskapet. 
 For at 321 skal betale mellomlegget mellom utbetaling fra Nav og full lønn må du ha jobbet i 321 i minimum ett år før du går ut i foreldrepermisjon.
 
+I forbindelse med fødsel har far rett på 2 uker fri (omsorgspermisjon). Denne permisjonen skal tas ut innenfor et tidsrom på innen ca 2 uker etter fødsel. Permisjonen kan deles opp, og det er helt i orden at man benytter for eksempel 1 til 2 dager knyttet til selve fødselen, gjenopptar arbeider og så tar resten av permisjonen når mor og barn kommer hjem fra sykehuset. 321 utbetaler full lønn for disse 2 ukene.
```

Comment:
Bytte de to første setningene med dette?

> I forbindelse med fødsel har far, medmor eller den som bistår mor under graviditeten rett til to uker permisjon. Permisjonen skal tas ut innenfor et tidsrom på 2 uker før til 2 uker etter fødsel.

 (Første setning fra https://www.altinn.no/starte-og-drive/arbeidsforhold/permisjoner-og-ferie/rettigheter-ved-graviditet-fodsel-adopsjon-og-omsorg-for-sma-barn/)

Reply:
Ja, enig!